src/main/java/org/ow2/proactive/catalog/service/BucketService.java

```diff
@@ -67,7 +67,7 @@
 @Transactional
 public class BucketService {
 
-    public static final String DEFAULT_BUCKET_OWNER = "public-objects";
+    public static final String DEFAULT_BUCKET_OWNER = "GROUP:public-objects";
```

Comment:
This should be : 

public static final String DEFAULT_BUCKET_OWNER = AuthorizationService.GROUP + ":public-objects";